tests/integration/ceph_upgrade_test.go

```diff
@@ -143,19 +143,19 @@ func (s *UpgradeSuite) TestUpgradeToMaster() {
 	require.NotEqual(s.T(), 0, numOSDs)
 
 	//
-	// Upgrade Rook from v1.1 to v1.2
+	// Upgrade Rook from v1.2 to v1.3
 	//
-	logger.Infof("*** UPGRADING ROOK FROM v1.1 to v1.2 ***")
-	s.gatherLogs(systemNamespace, "_before_1.2_upgrade")
-	s.upgradeToV1_2()
+	logger.Infof("*** UPGRADING ROOK FROM v1.2 to v1.3 ***")
+	s.gatherLogs(systemNamespace, "_before_1.3_upgrade")
+	s.upgradeToV1_3()
 
 	s.verifyOperatorImage(installer.Version1_2)
 	s.verifyRookUpgrade(numOSDs)
-	logger.Infof("Done with automatic upgrade from v1.1 to v1.2")
-	newFile := "post-upgrade-1_1-to-1_2-file"
+	logger.Infof("Done with automatic upgrade from v1.2 to v1.3")
+	newFile := "post-upgrade-1_2-to-1_3-file"
 	s.verifyFilesAfterUpgrade("", newFile, message, rbdFilesToRead, cephfsFilesToRead)
 	rbdFilesToRead = append(rbdFilesToRead, newFile)
-	logger.Infof("Verified upgrade from v1.1 to v1.2")
+	logger.Infof("Verified upgrade from v1.2 to v1.3")
 
 	//
 	// Upgrade from mimic to nautilus
```

Comment:
We can remove this upgrade step (lines 163-165) if we're starting with nautilus.